### ISSUE TITLE
Shadowbox error

### ISSUE THREAD

**Issue by AvC261** [Jigsaw-Code/outline-client#1295]
**Describe the issue**
after running the script a handfull of times (because i had some problesms with the firewall they are now solved) this started happening
sudo bash -c "$(wget -qO- https://raw.githubusercontent.com/Jigsaw-Code/outline-server/master/src/server_manager/install_scripts/install_server.sh)"
> Verifying that Docker is installed .......... OK
> Verifying that Docker daemon is running ..... OK
> Setting PUBLIC_HOSTNAME to external IP ...... OK
> Creating persistent state dir ............... OK
> Generating secret key ....................... OK
> Generating TLS certificate .................. OK
> Generating SHA-256 certificate fingerprint .. OK
> Writing config .............................. OK
> Starting Shadowbox .......................... FAILED
Unable to find image 'quay.io/outline/shadowbox:stable' locally
stable: Pulling from outline/shadowbox
cbdbe7a5bc2a: Pulling fs layer
8f3938f7d3bd: Pulling fs layer
6e3c12f5dc10: Pulling fs layer
ce0cb7a9eeee: Pulling fs layer
90580165d776: Pulling fs layer
1ed2bec38931: Pulling fs layer
93243a5b7df3: Pulling fs layer
7c7f63422259: Pulling fs layer
70c3ffa1c0bf: Pulling fs layer
bb6b8fb81cb5: Pulling fs layer
5a0f6aeaf89b: Pulling fs layer
53bbf77f5c86: Pulling fs layer
1ed2bec38931: Waiting
93243a5b7df3: Waiting
7c7f63422259: Waiting
70c3ffa1c0bf: Waiting
bb6b8fb81cb5: Waiting
5a0f6aeaf89b: Waiting
53bbf77f5c86: Waiting
ce0cb7a9eeee: Waiting
90580165d776: Waiting
cbdbe7a5bc2a: Verifying Checksum
cbdbe7a5bc2a: Download complete
6e3c12f5dc10: Verifying Checksum
6e3c12f5dc10: Download complete
cbdbe7a5bc2a: Pull complete
ce0cb7a9eeee: Verifying Checksum
ce0cb7a9eeee: Download complete
8f3938f7d3bd: Verifying Checksum
8f3938f7d3bd: Download complete
1ed2bec38931: Verifying Checksum
1ed2bec38931: Download complete
90580165d776: Verifying Checksum
90580165d776: Download complete
93243a5b7df3: Verifying Checksum
93243a5b7df3: Download complete
70c3ffa1c0bf: Download complete
7c7f63422259: Verifying Checksum
7c7f63422259: Download complete
bb6b8fb81cb5: Download complete
8f3938f7d3bd: Pull complete
6e3c12f5dc10: Pull complete
53bbf77f5c86: Download complete
ce0cb7a9eeee: Pull complete
90580165d776: Pull complete
1ed2bec38931: Pull complete
5a0f6aeaf89b: Verifying Checksum
5a0f6aeaf89b: Download complete
93243a5b7df3: Pull complete
7c7f63422259: Pull complete
70c3ffa1c0bf: Pull complete
bb6b8fb81cb5: Pull complete
5a0f6aeaf89b: Pull complete
53bbf77f5c86: Pull complete
Digest: sha256:e98e7673eed63bf4871e07c934a7694e592686860119f0d871e14f8ad56f45e5
Status: Downloaded newer image for quay.io/outline/shadowbox:stable
docker: Error response from daemon: error while creating mount source path '/opt/outline/persisted-state': mkdir /opt/outline: read-only file system.

Sorry! Something went wrong. If you can't figure this out, please copy and paste all this output into the Outline Manager screen, and send it to us, to see if we can help you.
Full log: /tmp/outline_logwhl

**Environment**
 - OS: [e.g. iOS]Ubuntu
 - NPM version (`npm --version`)8.5.1
 - Node.js version12.22.9

**Additional context**

**Comment by daniellacosse** [comment]
I don't think this helps too much but Node version should be lts/gallium. Make sure you have that and your dependencies are up to date (`npm run reset`)
